proxy/auth/aap.go

```
@@ -0,0 +1,175 @@
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	body, err := json.Marshal(UserInfoResponse{Username: userInfo.Results[0].Username})
+	if err != nil {
+		log.Warnf("Failed to marshall user info: %s", err.Error())
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	w.Write(body)
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)